examples/plotting/line_on_off.py

```diff
@@ -19,7 +28,7 @@
 checkbox = CheckboxGroup(labels=["Line 0", "Line 1", "Line 2"],
                          active=[0, 1, 2], width=100)
 
-callback = CustomJS(args=dict(l0=l0, l1=l1, l2=l2, checkbox=checkbox), code="""
+callback = CustomJS(args={"l0":l0, "l1":l1, "l2":l2,"checkbox":checkbox}, code="""
```

Comment:
I prefer to use `dict` for these cases. The names need to be proper identifiers, and using kwargs to `dict` helps reinforce that.